pkg/rancher-desktop/utils/_demo_marketplace_items.js

```diff
@@ -1,5 +1,31 @@
 export const demoMarketplace = {
   summaries: [
+    {
+      id:                    '',
+      name:                  'Open WebUI Extension',
+      containerd_compatible: true,
+      slug:                  'ghcr.io/rancher-sandbox/rancher-desktop-rdx-open-webui',
+      type:                  'extension',
+      publisher:             { name: 'SUSE LLC' },
```

Comment:
Even though we are not updating Epinio, shouldn't we still fix the company name?

Reply:
Agreed, Done.